src/schema/objects/columns.yaml

```diff
@@ -79,12 +90,14 @@ duration:
     - n/a
 filename:
   name: filename
+  display_name: Filename
   description: |
     Relative paths to files.
   type: string
   format: participant_relative
-group:
+channel_group:
```

Comment:
Our pattern has generally been `group__channel`, but I don't know if we need to adhere to that after reorganizing.

Reply:
```suggestion
group__channel:
```